storage/storage_helpers.cpp

```diff
@@ -20,20 +20,25 @@ bool IsDownloadFailed(Status status)
          status == Status::EUnknown;
 }
 
-bool IsEnoughSpaceForDownload(TMwmSize size)
+bool IsEnoughSpaceForDownload(TMwmSize mwmSize, TMwmSize maxMwmSize)
 {
-  // Mwm size is less than kMaxMwmSizeBytes. In case of map update at first we download updated map
+  // Mwm size is less than |maxMwmSize|. In case of map update at first we download updated map
   // and only after that we do delete the obsolete map. So in such a case we might need up to
   // kMaxMwmSizeBytes of extra space.
-  return GetPlatform().GetWritableStorageStatus(size + kMaxMwmSizeBytes) ==
+  return GetPlatform().GetWritableStorageStatus(mwmSize + maxMwmSize) ==
```

Comment:
I mean, the comment contradicts with the semantic. If we need to download new map and then remove old map, all what we need is ```size(new map)```, right?